st2common/tests/integration/test_register_content_script.py

```diff
@@ -43,7 +46,7 @@ def setUp(self):
         test_config.parse_args()
 
     def test_register_from_pack_success(self):
-        pack_dir = os.path.join(get_fixtures_packs_base_path(), "dummy_pack_1")
```

Comment:
In most tests, we actually calculate the fixture path every time it's used. By adding the boilerplate inside each fixture, that calculation only needs to be done in one place (DRY). Instead, we just need to use the imported `DUMMY_PACK_1_PATH` constant here.